locales/fr.json

```diff
@@ -1,95 +0,0 @@
-{
```

Comment:
Is it possible for you to use git mv instead of mv for moving template files so the git history on the files wont be lost? It seems that fr.json is deleted and created again.

Reply:
Jep. I'll try my best.

Reply:
done